lib/api/health.js

```diff
@@ -0,0 +1,118 @@
+'use strict';
+
+const Joi = require('joi');
+const tools = require('../tools');
+const { successRes } = require('../schemas/response/general-schemas');
+
+module.exports = (db, server, loggelf) => {
+    server.get(
+        {
+            path: '/health',
+            summary: 'Check the health of the API',
+            description: 'Check the status of the WildDuck API service, that is if db is connected and readable/writable, same for redis.',
+            tags: ['Health'],
+            validationObjs: {
+                requestBody: {},
+                queryParams: {},
+                pathParams: {},
+                response: {
+                    200: {
+                        description: 'Success',
+                        model: Joi.object({ success: successRes })
+                    },
+                    500: {
+                        description: 'Failed',
+                        model: Joi.object({ success: successRes, message: Joi.string().required().description('Error message specifying what went wrong') })
+                    }
+                }
+            }
+        },
+        tools.responseWrapper(async (req, res) => {
+            res.charSet('utf-8');
+
+            const currentTimestamp = Date.now() / 1000;
```

Comment:
This value is basically a float, but you are using it as a string. Maybe round it and then cast it into a string to prevent edge case surprises (eg. if for some value dividing by 1000 does not end up as a fixed number, but something like 12345.99999999...)

```
currentTimestamp = Math.round(Date.now() / 1000).toString();
```